uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/API/Handler/GenericLeaveFieldPermissionsTest.php

```diff
@@ -232,6 +240,9 @@ public function testProcessWhenUserIsAdmin() {
   }
 
   public function testProcessWillHideAccessibleFieldsWhenRowIdentifierIsAbsentEvenIfUserHasAccess() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'hide_label' => 1
+    ]);
```

Comment:
Wouldn't it make more sense to use `hide_label => 0` here, given the test is about checking that users will not be able to see something even if they have access to it?

Reply:
Good idea.